exercise_0/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.14)
+project(exercise_0)
+
+set(CMAKE_CXX_STANDARD 14)
+
+include_directories(characterCounter)
+include_directories(fileMode)
+include_directories(terminalMode)
```

Comment:
In modern CMake, `target_include_directories` is preferred. It allows you to set different include paths for different targets (executables, libraries etc). When you use `include_directories`, the whole project is affected.